### PR TITLE
global: adds claim mode.

### PR DESCRIPTION
**Pull request by kaplun**
* Adds the claim interface functionallity.
  (closes #23)

Signed-off-by: Panos Paparrigopoulos panos.paparrigopoulos@cern.ch

### CODE REVIEW

File: dist/templates/literature/facets.html [hunk 1]
```diff
@@ -1,20 +0,0 @@
-<div ng-init="order={'doc_type': 'Document type', 'subject': 'Subject', 'author': 'Author', 'experiment': 'Experiment'}">
```
Comment (jmartinm) [inline dist/templates/literature/facets.html:1]: don't think this file should disappear completely from dist/ folder

Comment (kaplun) [reply dist/templates/literature/facets.html:1]: Yeah connected with the other ticket.

File: dist/inspirehep-search.js [hunk 2]
```diff
@@ -100,6 +110,153 @@
...
+
+    function claimRecord(recordId) {
+      /**
+     * Claims the record in behalf of the logged-in author.
+     */
+      console.log('Claiming ' + recordId);
+      var body = {
+            recordId: recordId,
+            orcid_id: $scope.user_orcid_id
+            phonetic_block:
```
Comment (jmartinm) [inline dist/inspirehep-search.js:220]: empty?

File: dist/templates/literature/results.html [hunk 3]
```diff
@@ -53,6 +53,11 @@ <h2 class="record-brief-title">
           </span>
         </div>
       </div>
+      <div ng-if="vm.ClaimToggled" class="col-md-3 right-column">
+        <button ng-if="!vm.alreadyClaimed(record.id) && vm.isClaimable(record.id)" type="button" ng-click="vm.claimRecord(record.id)" class="btn btn-success btn-claim">Claim this record</button>
```
Comment (jmartinm) [inline dist/templates/literature/results.html:57]: Both buttons on click seem to do the same action `vm.claimRecord(record.id)` so what is the difference?

**Comment by jmartinm** [comment]
We will have to revive https://github.com/inspirehep/inspirehep-search-js/pull/23 once it is needed.

**Pull request closed without merging**